### PR TITLE
{lib}[GCCcore/11.2.0] ngtcp2 v0.7.0

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

requires:
* #16097
* #16086

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3151.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/b0dfcad32fbf8dd966799aa50d23e7ae for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/n/ngtcp2/ngtcp2-0.7.0-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    ('CUnit', '2.1-3'),
+]
+
+dependencies = [
+    ('OpenSSL', '1.1', '', True),
+    ('GnuTLS', '3.7.3'),
+    ('nghttp3', '0.6.0'),
+]
+
+osdependencies = [('gnutls-devel', 'gnutls-dev', 'libgnutls-devel')]
```
Comment (boegel) [inline easybuild/easyconfigs/n/ngtcp2/ngtcp2-0.7.0-GCCcore-11.2.0.eb:37]: @sassy-crick Why not depend on `GnuTLS` instead?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Tests on both `generoso` and `jsc-zen2` failed with:
```
One or more OS dependencies were not found: [('gnutls-devel', 'gnutls-dev', 'libgnutls-devel')]
```

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
eb-u2204 - Linux Ubuntu 22.04, x86_64, AMD Ryzen 9 3900X 12-Core Processor, Python 3.10.4
See https://gist.github.com/6057ed08187fda851bc1392344ee9fe4 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3144.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/4956081958d031816e24aedb23f23d5a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Pull request merged**